oauthenticator/azuread.py

```diff
@@ -32,12 +32,20 @@ class AzureAdOAuthenticator(OAuthenticator):
     def _tenant_id_default(self):
         return os.environ.get('AAD_TENANT_ID', '')
 
-    username_claim = Unicode(config=True)
+    username_claim = Unicode(config=True, help="Name of claim containing username")
 
     @default('username_claim')
     def _username_claim_default(self):
         return 'name'
 
+    user_groups_claim = Unicode(
+        config=True, help="Name of claim containing user group memberships"
+    )
+
+    @default('user_groups_claim')
+    def _user_groups_claim_default(self):
+        return 'groups'
```

Comment:
If you have a default that is fixed like this, you can have it as the first argument passed to the `Unicode` constructor above instead.